.gitignore

```diff
@@ -0,0 +1,6 @@
+.eslintcache
+build
+node_modules
+brave
```

Comment:
Note that `brave` will match `lib/whatever/brave/` too. Use `/brave` if you only want to exclude `brave/`

Reply:
Good point, I guess currently there is no such nested folder but will update to be on the safe side.

Reply:
Yeah, probably best to prepend all of these with `/`.

Reply:
Fixed here: https://github.com/brave/web-discovery-project/pull/12